### PR TITLE
Build fixes for kernel 6+

### PR DESCRIPTION
**Pull request by colinleroy**
This PR addresses two issues with kernels 6.1+ and 6.6+, it is a merge of both #49 and #52.

(btw, I see that very simple, acked PRs linger for months without being merged, and I want to indicate that if you folks cannot/don't want to maintain this useful tool anymore, I am OK to help)

### CODE REVIEW

**Comment by stappersg** [comment]
On Sun, Sep 15, 2024 at 03:03:19AM -0700, Colin Leroy-Mira wrote:
> This PR addresses two issues with kernels 6.1+ and 6.6+, it is a merge of both #49 and #52.

Please split into seperate commits.

Pretty please: Don't bother github.com references


> (btw, I see that very simple, acked PRs linger for months without
> being merged, and I want to indicate that if you folks cannot/don't
> want to maintain this useful tool anymore, I am OK to help)

Oops, I forgot that I have merge privilege for this git repository.
So thanks for the wake-up call.



Groeten
Geert Stappers
-- 
Silence is hard to parse

**Comment by stappersg** [comment]
It should be separate commits.

**Comment by colinleroy** [comment]
Hi!

> It should be separate commits.

So I've repushed #49 cleanly to #57, and @ssterling 's fix for kernel 6.6 is in #52.

HTH & thanks!
